ruby-challenges.rb

```diff
@@ -22,6 +49,9 @@
 beatles_album3 = 'Abbey Road'
 # Expected output: 'bby Rd'
 
+puts beatles_album1.delete 'aeiou'
```

Comment:
Delete is definitely my favorite choice for this problem.  Keep in mind you will want this to be a method so it is reusable.